### ISSUE TITLE
Zea.live -  Header disappearing and transparency

### ISSUE THREAD

**Issue by juanvelazco** [ZeaInc/zeainc.github.io#74]
If we scroll down, the header stays on top all the time.

![ZEA LIVE improvements 2](https://user-images.githubusercontent.com/67337724/88206103-663dea00-cc13-11ea-92c3-c62117634944.gif)

On the designs, the header is transparent, and it disappears when you scroll down.
https://xd.adobe.com/view/67e9f444-0e3c-4693-5359-7e4a1c73b02e-11d8/screen/45b0f9cc-9dcc-4ce0-9157-86c4077b1a0c/
![ZEA LIVE improvements 1](https://user-images.githubusercontent.com/67337724/88206320-aef5a300-cc13-11ea-9fd3-f2dcd4960114.gif)

**Comment by robhaines** [comment]
Yeah, the old site had this functionality. https://www.android.com/ is an example.

![ezgif com-video-to-gif](https://user-images.githubusercontent.com/49651841/88300601-4963ee00-ccc9-11ea-8394-748a6bf52e5a.gif)
